src/openrct2/interface/Viewport.cpp

```diff
@@ -952,7 +952,7 @@ void viewport_paint(
     uint32_t viewFlags = viewport->flags;
     uint32_t width = right - left;
     uint32_t height = bottom - top;
-    uint32_t bitmask = viewport->zoom >= 0 ? 0xFFFFFFFF & (0xFFFFFFFF * viewport->zoom) : 0xFFFFFFFF;
+    uint32_t bitmask = viewport->zoom >= ZoomLevel{ 0 } ? 0xFFFFFFFF & (0xFFFFFFFF * viewport->zoom) : 0xFFFFFFFF;
```

Comment:
```suggestion
    const uint32_t bitmask = viewport->zoom >= ZoomLevel{ 0 } ? 0xFFFFFFFF & (0xFFFFFFFF * viewport->zoom) : 0xFFFFFFFF;
```
I know we have this in at least two places, should move this at some point into a function.